namespaces/live.cloud-platform.service.justice.gov.uk/justice-gov-uk-production/resources/rds-tmp.tf

```diff
@@ -0,0 +1,68 @@
+module "rds_tmp" {
+  source        = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=7.0.0"
+  vpc_name      = var.vpc_name
+  team_name     = var.team_name
+  business_unit = var.business_unit
+  application   = var.application
+  namespace     = var.namespace
+  is_production = var.is_production
+
+  # turn off performance insights
+  performance_insights_enabled = false
+
+  # general options
+  db_engine                   = "mariadb"
+  db_engine_version           = "10.11.6"
+  rds_family                  = "mariadb10.11"
+  db_instance_class           = "db.t4g.medium"
+  db_allocated_storage        = "5"
+  environment_name            = var.environment
+  infrastructure_support      = var.infrastructure_support
+  allow_major_version_upgrade = "false"
+  deletion_protection = true
```

Comment:
Do we need this?

Reply:
Oh ... nope.

Reply:
... it might cause us problems once we have the dump and want to remove it. I.e. multiple PRs to CP.